src/index.ts

```diff
@@ -1,3 +1,7 @@
+import { escapeRegExp } from "./utils";
+
+export { escapeRegExp } from "./utils";
```

Comment:
why export from here when you're already doing it from within `utils.ts`?

Reply:
To use it in a project you would need to do `import { escapeRegExp } from 'aracari/dist/utils';` to use it in utils. With this export you can use `import { escapeRegExp } from 'aracari';`